packages/genui/a2ui-playground/src/pages/BenchPage.css

```
@@ -861,6 +861,340 @@
+  line-height: 1.25;
+}
+
+.benchScreenshotPlaceholder span {
+  display: -webkit-box;
+  overflow: hidden;
+  color: var(--geist-secondary);
+  font-size: 12px;
+  line-height: 1.4;
+  overflow-wrap: anywhere;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)